### PR TITLE
Update Native Styling Getting Started

### PR DESCRIPTION
**Pull request by ConnorLand**
Per Danny's feedback, this document has been updated:

* _rosybrown-code.png_ has been replaced with a small code snippet, allowing readers to copy and paste
* A sentence concerning Mendix Atlas UI has been removed because it incorrectly supplies web classes to native mobile readers
* Links to the Native Styling Reference Guide have been added in both the Introduction and Read More sections

### CODE REVIEW

File: content/howto/mobile/how-to-use-native-styling.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags: ["styling", "design", "classes", "native"]
 
 ## 1 Introduction
 
-With Mendix 8, you have the capacity to alter design properties with Mendix Studio Pro. Furthermore, because all native mobile styling is written in JavaScript, you have new ways of applying your styling customizations.
+With Mendix 8, you have the capacity to alter design properties with Mendix Studio Pro. Furthermore, because all native mobile styling is written in JavaScript, you have new ways of applying your styling customizations. For more details on native styling, class names, and widget styling, see the [Native Styling Reference Guide](/refguide/native-styling-refguide).
```
Comment (Adam-Dupaski) [inline content/howto/mobile/how-to-use-native-styling.md]: "Native Styling Reference Guide" is not the title of this doc. Please update the title in your cross-reference.

File: content/howto/mobile/how-to-use-native-styling.md [hunk 2]
```diff
@@ -177,5 +185,6 @@ Congratulations! By completing this how-to, you have learned how to alter a styl
 ## 8 Read More
 
 * [Native Styling](native-styling)
+* [Native Styling Reference Guide](/refguide/native-styling-refguide)
```
Comment (Adam-Dupaski) [inline content/howto/mobile/how-to-use-native-styling.md]: Again, not the correct title of this Ref Guide doc in the cross-reference. If you need to differentiate between titles that are the same between Ref Guide and How-to's, consider these options:
1) Make one title different. ;) Consider renaming "Native Styling" how-to sub-category to "Implement Native Styling" (this is the best choice)
2) Refer to "Native Styling How-to's", as that is a collection of how-to's; if that was a single doc and not a parent page/collection, then the title would be "How to..."

Comment (ConnorLand) [reply content/howto/mobile/how-to-use-native-styling.md]: Option 1 taken. Title has been changed throughout docs.

File: content/howto/mobile/how-to-use-native-styling.md (hunk 1 above)
Comment (ConnorLand) [reply content/howto/mobile/how-to-use-native-styling.md]: Updated.

File: content/howto/mobile/how-to-use-native-styling.md (hunk 2 above)
Comment (ConnorLand) [reply content/howto/mobile/how-to-use-native-styling.md]: Option 1 taken. Thank you.

Comment (Adam-Dupaski) [reply content/howto/mobile/how-to-use-native-styling.md]: "Native Styling Reference Guide" still needs to be changed

Comment (ConnorLand) [reply content/howto/mobile/how-to-use-native-styling.md]: Changed.

**Pull request merged**